client/components/user-select/select.directive.js

```diff
@@ -7,7 +7,9 @@ angular.module('probedock.userSelect').directive('userSelect', function() {
       organization: '=',
       modelObject: '=',
       modelProperty: '@',
-      prefix: '@'
+      prefix: '@',
+      filterByLabel: '@',
+      filterAllLabel: '@'
```

Comment:
These scope argument names don't sound very intuitive to me considering how they're used. I would prefer `label` and `placeholder`, or `fieldLabel` and `fieldPlaceholder`, for example. It better expresses their function.